index.html

```diff
@@ -8,7 +8,7 @@
     <style>
```

Comment:
hey! move styles to files! IMMEDIATELY

Reply:
true... but you could have done it before! MOVE YOUR FINGERS!!

Reply:
Sure. I'll do it after the break!